### ISSUE TITLE
Manual ellipse in Distance Direction has not feedback line on minor axis

### ISSUE THREAD

**Issue by lfunkhouser** [Esri/solutions-webappbuilder-widgets#733]
_From @mfunk on July 28, 2016 20:59_

In testing the manual ellipses in Distance Direction on deployment server the second (minor) axis line has no visual feedback graphic between center point and cursor.

![image](https://cloud.githubusercontent.com/assets/3093182/17229182/699ba0de-54cb-11e6-9573-ef39454573c7.png)

The cursor moves, and if you click the ellipse will be created using the point. The problem is that you can't see the line that is being used as the axis length.

_Copied from original issue: Esri/military-tools-webappbuilder#91_

**Comment by lfunkhouser** [comment]
_From @pshowalter on July 28, 2016 21:27_

It seems to work in Chrome.  
Not working in IE or  FF.
Browser specific issue?

**Comment by lfunkhouser** [comment]
_From @kgonzago on July 29, 2016 13:8_

@mfunk, Like @pshowalter mentioned, which browser were you testing the widget with?

**Comment by lfunkhouser** [comment]
_From @kgonzago on July 29, 2016 15:37_

Confirmed it is an issue with IE and FF. Will work on it

**Comment by lfunkhouser** [comment]
_From @adgiles on December 12, 2016 13:14_

@kgonzago @nfeuerstein @mfunk 

The widget just kept returning a type error when calculating the minor axis length using the geometry engine geodesic length function, but only in FF and IE. I could not work out why the browser was doing this as it was happy with the same function call for the major axis length.

I tried changing the unit value 9001 to 'meters' and also tried setting it as a variable and passing it in to the function but no change in IE and FF. 

In the end I changed the function to use the geodesicLengths in geodesicUtils:

minorAxisLength = geodesicUtils.geodesicLengths([this._minGraphic.geometry], Units.METERS);

This worked straight away, whilst fixing this issue I also added a handler to handle the user double clicking when drawing the first axis. It this happens it now clears the previous graphics and starts over, before it left lots of random graphics on the screen
